.github/stale.yml

```diff
@@ -0,0 +1,20 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 45
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - status: status: willfix
+# Label to use when marking an issue as stale
+staleLabel: status: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This pull request has been automatically marked as stale because it has not 
```

Comment:
Here and elsewhere. YAML lint findings, drop trailing whitespace please!

Reply:
Will work around them all